pyserini/encode/_uniir.py

```diff
@@ -136,3 +139,119 @@ def encode(
             corpus_embeddings = corpus_embeddings.astype('float16') if use_fp16 else corpus_embeddings
 
         return corpus_embeddings
+
+
+class UniIRQueryEncoder(UniIREncoder):
+    def __init__(
+        self,
+        encoder_dir: str,
+        device="cuda:0",
+        l2_norm=False,
+        instruction_config=None,
```

Comment:
I don't see a yaml file for this config added anywhere. I was expecting something similar to uniir model configs that you added.